### ISSUE TITLE
[skip to content] Users unable to skip to content on developer site

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#193]
URL: https://developers.openreferraluk.org/

Users are unable to skip to content as this functionality is not present.

**Comment by UsmanAfzal** [comment]
Checked and approved
